tests/scripts/booking/cancel_expired_bookings_test.py

```diff
@@ -0,0 +1,75 @@
+from datetime import datetime
+from datetime import timedelta
+
+import pytest
+from pytest import approx
+
+from pcapi.core.bookings.factories import BookingFactory
+from pcapi.core.bookings.models import BookingCancellationReasons
+from pcapi.core.offers.factories import ProductFactory
+from pcapi.models import offer_type
+from pcapi.repository import repository
+from pcapi.scripts.booking.cancel_expired_bookings import cancel_expired_bookings
+
+
+@pytest.mark.usefixtures("db_session")
+class CancelExpiredBookingsTest:
+    def test_old_non_subscription_thing_booking(self, app) -> None:
+        two_months_ago = datetime.utcnow() - timedelta(days=60)
+        book = ProductFactory(type=str(offer_type.ThingType.LIVRE_EDITION))
+        old_book_booking = BookingFactory(stock__offer__product=book, dateCreated=two_months_ago)
+
+        cancel_expired_bookings()
+
+        assert old_book_booking.isCancelled
+        assert old_book_booking.cancellationDate.timestamp() == approx(datetime.utcnow().timestamp())
+
+    def test_new_non_subscription_thing_booking(self, app) -> None:
```

Comment:
Et ici je lirait bien
```suggestion
    def should_not_cancel_new_non_subscription_thing_booking(self, app) -> None:
```

Reply:
L'avantage que tu cibles directement les cas qui sont concernés par la règle (ceux qui doivent être annulés) et ceux qui ne le sont pas.